### PR TITLE
Add election information section

### PR DESCRIPTION
**Pull request by alexsoble**
What do y'all think?

### CODE REVIEW

**Comment by jeancochrane** [comment]
Thanks for this @alexsoble! Excited to take a look tonight.

**Comment by jeancochrane** [comment]
Left some comments in our Site Text doc, as per [the workflow Bea suggested](https://github.com/jeancochrane/chano4mayor/pull/15#issuecomment-287083941):

[https://docs.google.com/document/d/1IX2bi34y2y00EOO6SrGxe55iVo-GhdIweq908Y1geuQ/edit?usp=sharing](https://docs.google.com/document/d/1IX2bi34y2y00EOO6SrGxe55iVo-GhdIweq908Y1geuQ/edit?usp=sharing)

Happy to make these changes if you agree with them!

**Comment by alexsoble** [comment]
Oooooh, sorry for messing up on the workflow part!  I agree that @beamalsky's suggested workflow is very good, just forgot about.  

And @jeancochrane -- I **love** your suggested copy from the Google Doc. Left some notes there to that effect as well. 😊

**Comment by kalilsn** [comment]
@alexsoble I think your workflow is good too! Once the site is live I think we should all get in the habit of using pull requests to make changes, and waiting for someone else to review them before merging (so we don't carelessly break anything).

File: index.html [hunk 1]
```diff
@@ -121,11 +121,43 @@
...
                         </div>
             </div>
         <div class="section section-blue tweets">
             <div class="wrap">
+                <h1>Hey Chicago,</h1>
+
+                <p>Here are some information you can use if you like the idea of Chance for Mayor, or someone who would fight for the things Chance fights for:</p>
+
+                <br/>
+                <br/>
```
Comment (kalilsn) [inline index.html]: @alexsoble If these `<br>`s are just for spacing, can we just do it with css instead? It'll make our html easier to maintain in the long run + if each paragraph is wrapped in a `<p>` tag the spacing will stay consistent with the rest of the site. Sorry to nitpick, just wanna keep it clean in the long run!

Comment (alexsoble) [inline index.html]: Yeah good catch @kalilsn !

**Comment by kalilsn** [comment]
Ok, I've added our changes but let me know if I missed anything! I also did a few small style things and some cleaning of the markup.

If anyone has any suggestions about style, I'm all ears. I think the faq questions need a visual way to separate them from the answers. Right now it's italics, which is ok, but not great. Underlining looked better but unfortunately that was the exact same style as links. So I'm looking for ideas for changing either one!

**Comment by jeancochrane** [comment]
Looks good! I noticed a missing sentence and a missing link, so I went ahead and added those in.

I think @beamalsky also wanted to have an explicit header for this section ("About the 2019 Mayoral Election"). Do we still want that?

If we went ahead and brought in a secondary header font per #26 , it might also allow us to better distinguish the questions from the answers. Once we pull these changes in I'll make that PR and we can try it.

**Comment by beamalsky** [comment]
Thanks for putting this together Kalil! 

I still think we should have the election info at the bottom (see #24) and keep the About the 2019 Mayoral Election header, maybe centered directly above the first question.

I'd like to make the Q&A sections narrower than the body text sections (indented, effectively), and keep the "Q:" and "A:" text that's in the Site text doc. If at all possible I think we should use avoid having the questions be slightly bigger than the answers, and Jean's second font seems like a good solution.

**Comment by beamalsky** [comment]
Also @jeancochrane, @kalilsn cut that sentence intentionally! He's right that we don't need to direct people to the closed link.

**Comment by kalilsn** [comment]
@beamalsky is right that I removed that sentence intentionally, but @jeancochrane did change the link to a more appropriate one. I still don't think it's helpful to send people to that page while it's not available but I'm not too concerned.

I think indentation is good, but I'm not crazy about the Q. and A. text on the site. How would you feel about showing/hiding the answer text on click and adding some kind of `+` or other bullet icon to the questions?

I'm not really confident people will scroll past the twitter section if we put it before this. It's already a lot of text for people to read and I think the voting info is way more important than the tweets. On mobile that's an especially big issue, since it's basically impossible to scroll past the twitter widget.

File: index.html [hunk 2]
```diff
@@ -137,9 +137,9 @@
...
-                <p>If you don’t have those forms of ID but qualify to vote (see above), you can still register by printing and filling out <a href="http://app.chicagoelections.com/documents/general/NVRA-Form275-English.pdf" target="_blank">this form</a>. You may also need to bring identification when you vote—more info <a href="http://www.chicagoelections.com/en/when-you-need-id-to-vote.html" target="_blank">here</a>.</p>
+                <p>If you have an Illinois state ID or driver’s license, you can register online nearer to the election. Registration is down right now, but you can find it <a href="https://ova.elections.il.gov/RegistrationClosed.aspx">here</a> and we’ll update this site when it’s back up. If you don’t have those forms of ID but qualify to vote (see above), you can still register by printing and filling out <a href="http://app.chicagoelections.com/documents/general/NVRA-Form275-English.pdf" target="_blank">this form</a>. You may also need to bring identification when you vote—more info <a href="http://www.chicagoelections.com/en/when-you-need-id-to-vote.html" target="_blank">here</a>.</p>
```
Comment (kalilsn) [inline index.html]: I think we should add a `target="_blank"` on this link for consistency.

File: index.html [hunk 3]
```diff
@@ -137,9 +137,9 @@
...
                 <h2>How do I know if I'm registered?</h2>
-                <p>You can check if you're registered to vote in Chicago elections with this tool.</p>
+                <p>You can check if you're registered to vote in Chicago elections with <a href="https://ova.elections.il.gov/RegistrationLookup.aspx">this tool</a>.</p>
```
Comment (kalilsn) [inline index.html]: Same as above :)

**Comment by alexsoble** [comment]
Agreed with @kalilsn that folks are unlikely to scroll past the Twitter embed.

I also like the idea of show/hide for Q&A answers.

**Comment by alexsoble** [comment]
I have some toggling-sections CSS I can quickly reuse here from another project...

**Comment by beamalsky** [comment]
I like the idea of adding toggles to the questions a lot! I still like the style of including Q/A but I don't feel strongly about it if others want to keep it off

**Comment by beamalsky** [comment]
Talked to Jean a bit about section order—could we have a phone call or something about it? I really feel like it reads better with the Twitter embed after the Chance section, and at this point we're pushing the amount that anyone is likely to read all the way through. I agree it's difficult/unlikely that people will scroll through the Twitter embed, but I'd rather they stop on that than on the election info at this point. (ie, if we're maintaining this as the election gets closer I'd feel differently, but for now the Twitter hype and connecting this to what other people are saying is more important I think.)

**Comment by alexsoble** [comment]
Did some playing around with both orderings . . . @beamalsky I am cool with Twitter embed after the Chance section for sure.

**Comment by alexsoble** [comment]
# GIF (0b3fa6db906930a218bf6c9ed5b8943aa313d4c7)

faq

**Comment by alexsoble** [comment]
@kalilsn -- 0b3fa6db906930a218bf6c9ed5b8943aa313d4c7 introduces a couple of `<br/>`s, sorry! Using CSS kept breaking the wrap, wasn't sure how to get it done any other way.

**Comment by beamalsky** [comment]
so beautiful, love the flip :')

**Comment by jeancochrane** [comment]
Looks great! I'm going to merge this in, since I'd like to do some work on top of it that's unrelated to the third section.

**Pull request merged**